### DISCUSSION TITLE
How do I make /docs as the landing page?

### DISCUSSION THREAD

**Discussion by SenPr** [colinwilson/lotusdocs#25] [Q&A]
Is there a way to make /docs as the landing page, instead of having to redirect?

**Comment by colinwilson** [comment]
Sorry. Not possible at the moment, not without a lot of changes. But it's something I've thought about and how best to implement.

**Comment by fenglisch** [comment]
This is a workaround: https://github.com/colinwilson/lotusdocs/issues/170
